src/test/kotlin/de/smartsquare/squit/task/SquitTestTaskSpek.kt

```diff
@@ -369,4 +372,35 @@ object SquitTestTaskSpek : SubjectSpek<Path>({
             }
         }
     }
+
+    given("a test project with an error response code") {
```

Comment:
Great test! Can we have another one where we assert that the build actually fails because of a `statusCode` mismatch?

Also extending the existing tests with some assertions would be very valuable. For example the `SquitRequestTaskSpek` could check if a valid `response_info.json` file was produced in addition to the `meta.json`

Reply:
Ok